packages/fondue/src/components/RichTextEditor/Plugins/MarkPlugin/SubscriptPlugin/SubscriptButton.tsx

```diff
@@ -1,22 +1,22 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { getTooltip } from '@components/RichTextEditor/helpers/getTooltip';
-import { MARK_SUPERSCRIPT, MarkToolbarButton, getPluginType } from '@udecode/plate';
-import { ButtonWrapper, IconStylingWrapper, buttonStyles, getButtonClassNames } from '../helper';
-import { PluginButtonProps } from '../types';
+import { MARK_SUPERSCRIPT, getPluginType } from '@udecode/plate';
+import { ButtonWrapper, IconStylingWrapper } from '../../helper';
+import { PluginButtonProps } from '../../types';
 import IconSubscript16 from '@foundation/Icon/Generated/IconSubscript16';
 import { getHotkeyByPlatform } from '@components/RichTextEditor';
+import { MarkToolbarButton } from '@components/RichTextEditor/Plugins/MarkPlugin/MarkToolbarButton';
 
 export const SubscriptButton = ({ editor, id }: PluginButtonProps) => (
     <ButtonWrapper id={id}>
         <MarkToolbarButton
             tooltip={getTooltip(`Subscript\n${getHotkeyByPlatform('Ctrl+.')}`)}
-            type={getPluginType(editor, id)}
+            key={id}
```

Comment:
Also here `key`, i think we can get rid of them

Reply:
Removed